tests/components/matter/test_sensor.py

```diff
@@ -92,6 +92,16 @@ async def eve_energy_plug_patched_node_fixture(
     )
 
 
+@pytest.fixture(name="eve_weather_sensor_node")
+async def eve_weather_sensor_node_fixture(
```

Comment:
Please don't duplicate the same fixture in different modules. Put it in the `conftest.py` module instead. Those fixtures can be accessed by the whole test package.